From 21de20cf88fdada442d91c6a446690d0c19d1936 Mon Sep 17 00:00:00 2001
Message-Id: <21de20cf88fdada442d91c6a446690d0c19d1936.1358959439.git.minovotn@redhat.com>
In-Reply-To: <4b2df53c087cd9df02d66686da40ae4f600f3904.1358959439.git.minovotn@redhat.com>
References: <4b2df53c087cd9df02d66686da40ae4f600f3904.1358959439.git.minovotn@redhat.com>
From: Markus Armbruster <armbru@redhat.com>
Date: Wed, 23 Jan 2013 15:52:29 +0100
Subject: [PATCH 3/7] Revert "hw/ac97: the volume mask is not only 0x1f"

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1358956353-15195-4-git-send-email-armbru@redhat.com>
Patchwork-id: 47664
O-Subject: [RHEL-6.4 PATCH qemu-kvm 3/7] Revert "hw/ac97: the volume mask is not only 0x1f"
Bugzilla: 884253
RH-Acked-by: Ademar de Souza Reis Jr. <areis@redhat.com>
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>

This reverts commit 529e1a5e2595006a1d42af9148f1a5dae9d4c7e0.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
---
 hw/ac97.c | 1 +
 1 file changed, 1 insertion(+)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hw/ac97.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hw/ac97.c b/hw/ac97.c
index 6552da3..8dd4f88 100644
--- a/hw/ac97.c
+++ b/hw/ac97.c
@@ -114,6 +114,7 @@ enum {
 #define EACS_VRA 1
 #define EACS_VRM 8
 
+#define VOL_MASK 0x1f
 #define MUTE_SHIFT 15
 
 #define REC_MASK 7
-- 
1.7.11.7

